packages/components/src/components/amount-input/amount-input.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import Input from '../input';
+
+type TAmountInput = {
+    currency: string;
+    decimalPoints?: number;
+    disabled?: boolean;
+    initialValue?: number;
+    label?: React.ReactNode;
+    maxDigits: number;
+};
+
+const AmountInput = ({
+    currency,
+    decimalPoints = 2,
+    disabled = false,
+    initialValue = 0,
+    label,
+    maxDigits,
+}: TAmountInput) => {
+    const [value, setValue] = useState(initialValue);
+    const [focus, setFocus] = useState(false);
+
+    const displayNumber = (number: number) => number.toLocaleString('en-US', { minimumFractionDigits: decimalPoints });
```

Comment:
@rostislav-deriv I believe that `en-US` should be based on the user's selected language as formating is a bit different in languages like Spanish (as we experienced it before in P2P)